sites/platform/src/create-apps/app-reference.md

```diff
@@ -375,31 +377,61 @@ applications:
 ```
 {{% /version/specific %}}
 
-For examples of how to set up a `service` mount, see the dedicated [Network Storage page](/add-services/network-storage.md).
-
 The accessibility to the web of a mounted directory depends on the [`web.locations` configuration](#web).
 Files can be all public, all private, or with different rules for different paths and file types.
 
 Note that when you back up an environment,
 the mounts on that environment are backed up too.
 
 Also, mounted directories aren't deleted when they're removed from `{{< vendor/configfile "app" >}}`.
-The files still exist on disk until manually removed (or, for `tmp` mounts, until the app container is moved to another host during a maintenance operation).
+The files still exist on disk until manually removed.
+Or, for `tmp` mounts, **until the app container is moved to another host during a maintenance operation.**
 
 {{% version/only "2" %}}
 
-### Mounts, instances, and Network Storage
+### Data sharing through mounts
```

Comment:
I'm not a fan of the title. I don't suggest anything here since I don't have a proper idea.